src/AFQMC/config.0.h

```diff
@@ -27,12 +27,18 @@
 #define PsiT_IN_SHM
 
 // guard with directive that checks if boost version is >=1.65
-#include <boost/version.hpp>
-#if BOOST_VERSION >= 106500
-#include <boost/stacktrace.hpp>
-#define print_stacktrace std::cout << boost::stacktrace::stacktrace();
+// uncomment to enable stacktrace
+//#define ENABLE_STACKTRACE
+#if defined(ENABLE_STACKTRACE)
+ #include <boost/version.hpp>
```

Comment:
Will putting `#include <boost/version.hpp>` inside ENABLE_STACKTRACE potentially causes problems for multi?
How about
```
#if BOOST_VERSION >= 106500 && defined(ENABLE_STACKTRACE)
```

Reply:
yeah, that's better